app/services/uuid_reporter.rb

```diff
@@ -0,0 +1,134 @@
+class UuidReporter
```

Comment:
I think it's good to test the happy path for this class as a Rake task, but I think it would be clearer to unit test it on its own

Reply:
@zachmargolis do you mean other than the tests in `spec/services/uuid_reporter_spec.rb`?

Reply:
No, I hadn't seen that file when I searched 🤦

Reply:
Ha no worries